src/scripts/main.js

```diff
@@ -1,8 +1,20 @@
 'use strict';
 
 const people = require('./lib/people');
+const dashboard = document.querySelector('.dashboard ');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+people.forEach(person => {
+  const tableRow = document.createElement('tr');
+  const { namee, sex, born, died } = person;
```

Comment:
It seems like you were having a problem with `name` variable. But the thing is, person object does not have `name `property, so use alias.

```suggestion
  const { name: personName, sex, born, died } = person;
```